src/bug_report.rs

```diff
@@ -17,7 +17,27 @@ pub fn create() {
         starship_config: get_starship_config(),
     };
 
-    let link = make_github_issue_link(environment);
+    let report = get_report(environment);
+
+    if print {
+        println!("{}", report);
+        return;
+    }
+
+    // Drop to system editor in a temp file
+    let report = match utils::edit_temp(report) {
+        Ok(r) => r,
+        Err(e) => {
+            eprintln!("Error editing bug report: {}", e);
+            return;
+        }
+    };
```

Comment:
Perhaps ask for confirmation before sending? Some editors might not wait for exit like `code` without `--wait`.

A flag for skipping editing in general might be good too (and reference it in relevant error messages).